rel/files/ejabberd.cfg

```diff
@@ -473,11 +473,10 @@
 %%% ==================================
 %%% POOLS OF OUTGOING HTTP CONNECTIONS
 %%% ==================================
-%%
-%% {http_connections, [{conn1, [{server, "http://server.com:8080"},
-%%                              {pool_size, 50}]}
-%%                    ]}.
 
+{http_connections, [{http_pool, [{server, "http://localhost:8000"},
```

Comment:
I don't think we want to enable such a pool by default - this is just an example.